commitlint.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: ['@commitlint/config-angular'],
+  rules: {
+    'subject-max-length': [1, 'always', 72],
+    'scope-empty': [1, 'never'],
```

Comment:
What does `never` mean? Does it mean it's not optional? Because the scope can be empty, for example if you change something like the README or CI or build-related things. The `app` scope is only meant to be used when there's a change for the demo app that is deployed to GitHub Pages.

Reply:
I'm new to commitlint as well 😅. Here are the [possible rules](https://github.com/marionebl/commitlint/blob/master/docs/reference-rules.md)

This is an excerpt from there:

> Level [0..2]: 0 disables the rule. For 1 it will be considered a warning for 2 an error.
Applicable always|never: never inverts the rule.
Value: value to use for this rule.

tbh, the `always` vs `never` is not that clear to me as well. But with the current configuration it's definitely possible to issue an empty scope. I've seen you use that so I made sure it's possible. It will only issue a warning basically.